pkg/metricscollector/opentelemetry.go

```diff
@@ -246,3 +246,28 @@ func getScalerMeasurementOption(namespace string, scaledObject string, scaler st
 		attribute.Key("metric").String(metric),
 	)
 }
+
+func (o *OtelMetrics) RecordScalerPaused(namespace string, scaledObject string, active bool) {
+	activeVal := 0
+	if active {
+		activeVal = 1
+	}
+
+	opt := api.WithAttributes(
+		attribute.Key("namespace").String(namespace),
+		attribute.Key("scaledObject").String(scaledObject),
+	)
+
+	cback := func(ctx context.Context, obsrv api.Float64Observer) error {
+		obsrv.Observe(float64(activeVal), opt)
+		return nil
+	}
+	_, err := meter.Float64ObservableGauge(
+		"keda.scaledobject.paused.total",
+		api.WithDescription("Total number of scaled objects currently in a paused state"),
+		api.WithFloat64Callback(cback),
+	)
```

Comment:
I know that it's how it's done in other gauges, but I'm not totally sure if we have to create an observable gauge on each execution. 
I have quickly reviewed the SDK but I haven't seen anything yet. Do you know about this @zroubalik ?

Reply:
I agree, we should try to find a better solution if there's any. Maybe try to ask in OTEL community or try to find suggested examples.

Reply:
@zroubalik I'm yet to figure out the proper way to tackle this issue. 
I think maybe we should create a separate issue for this. WDYT?

Reply:
okay, that's fair. Please open another issue and provide details about the problem, so we can track it and don't forget about it. Thanks!

Reply:
Alright Let me do that now.

Reply:
Please put a link to this specific conversation to the issue. This link: https://github.com/kedacore/keda/pull/5045/files/486ca2f1b1dd59d60cfbb86f5b5829264f8cd697#r1348641815

Reply:
Alright

Reply:
Done here is the issue 
https://github.com/kedacore/keda/issues/5101